### ISSUE TITLE
Types missing on bigquery.query => bigquery.ITableDataList

### ISSUE THREAD

**Issue by jakobskytte** [googleapis/nodejs-bigquery#931]
There are keys that are being returned when calling bigquery.query that doesn't exist on the typescript interface. Eg. schema is being returned when I call `bigquery.query(query, { maxResults: 1 })` but it doesn't exist on the type bigquery.ITableDataList.

#### Environment details

  - Node.js version: 14.5.4
  - `@google-cloud/bigquery` version: 5.5.0

#### Steps to reproduce

```ts
const queryResponse = await bigquery.query(query, { maxResults: 1 })
```
![Capture](https://user-images.githubusercontent.com/25202811/108611204-879aca80-73dc-11eb-87b5-ce90450f8799.PNG)

![Capture2](https://user-images.githubusercontent.com/25202811/108611225-c3ce2b00-73dc-11eb-925c-9ce6c226ee37.PNG)

**Comment by syJSdev** [comment]
I think this bug happens on v5.5.0. but fixed on v5.6.1.
Why don't you release v5.6.1 to npm? @steffnay

**Comment by steffnay** [comment]
5.7.0 is on npm, so this should be fixed now!
